<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>uiSingleServerWidget</class>
 <widget class="QWidget" name="uiSingleServerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>40</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>uiSingleServerWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>Server</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_Port">
       <property name="prefix">
        <string>Port: </string>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_Timeout">
       <property name="prefix">
        <string>Time out: </string>
       </property>
       <property name="maximum">
        <number>99999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_Address"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
